### DISCUSSION TITLE
I'm currently using Lottie version 2.5.3 with Objective-C.

### DISCUSSION THREAD

**Discussion by lkh10407** [airbnb/lottie-ios#2361] [Q&A]
The recent requirement for including privacyInfo isn't covered in this version, and due to the current code structure, updating to the latest version is also challenging. I need to incorporate privacyInfo into version 2.5.3. What approach can I take?
Due to Apple's policy, I need to have privacyInfo included by May 1st. Please help me

**Comment by calda** [comment]
We don't provide official support to the old Objective-C code, but if you'd like you can post a pull request on the [`lottie/objectiveC`](https://github.com/airbnb/lottie-ios/tree/lottie/objectiveC) branch that copies the privacy manifest over from `master`.

**Comment by lkh10407** [comment]
Thank you for answering.
After following your advice, I posted a pull request and checked the file, then I reinstalled it.

 I used the following command:
pod 'lottie-ios', :git => 'https://github.com/airbnb/lottie-ios.git', :branch => 'lottie/objectiveC'

However, when I reinstalled the pod file, it was not created. I could confirm that it is included in the Lottie folder when searching on git. I also attempted to install it via spm, but the installation was unsuccessful. Could you please check?

**Reply by shijian0** [reply]
me too

**Comment by TigerWf** [comment]
https://github.com/airbnb/lottie-ios/pull/2367
I left a message for this PR. The problem seems to be that he didn't change 'lottie-ios.podspec'. Just like this: add '**s.r esource_bundles = {" XXXXX "= >" XXX/PrivacyInfo. Xcprivacy"}**'

**Reply by LoveEunice** [reply]
@TigerWf Why is there still no PrivacyInfo.xcprivacy after I pod update

**Reply by TigerWf** [reply]
There is something wrong with the PR he posted, which only adds 'PrivacyInfo.xcprivacy', but it is not associated in 'lottie-ios.podspec'.  https://github.com/airbnb/lottie-ios/pull/2367
